trame_vtklocal/module/protocol.py

```diff
@@ -43,10 +60,20 @@ def unregister_widget(self, root_obj, dep_obj):
         if root_id in self._widgets:
             self._widgets[root_id].discard(dep_id)
 
-    def update(self, push_camera=False, **_):
+    def update(self, push_camera=False, obj_to_update=None, **_):
         self._push_camera = push_camera
 
-        self.vtk_object_manager.UpdateStatesFromObjects()
+        if API_NO_IDS_UPDATE:  # <= 9.4.2
+            self.vtk_object_manager.UpdateStatesFromObjects()
+        else:  # > 9.4.2
```

Comment:
should this comment be `<= 9.4.20250505`?

Reply:
The API_NO_IDS_UPDATE definition capture the proper MR and test in case it truly matter